### PR TITLE
Personal website project

### PR DESCRIPTION
**Pull request by KhanyiMM**
Related issues: [please specify]

## Description:

What are you up to? Fill us in :)

## I solemnly swear that:

- [ ] I ran the hugo server and looked at my changed in the browser with my own eyes
- [ ] I ran the linter and there were no errors
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code

### CODE REVIEW

File: content/projects/first-personal-website/_index.md [hunk 1]
```diff
@@ -2,19 +2,19 @@
 _db_id: 220
 content_type: project
 flavours:
-- none
+  - none
```
Comment (ry-oc) [inline content/projects/first-personal-website/_index.md:5]: Why are you editing the spacing?

File: content/projects/first-personal-website/_index.md [hunk 2]
```diff
@@ -2,19 +2,19 @@
...
 prerequisites:
   hard:
-  - topics/free-code-camp/responsive-web-design/1-basic-html-and-html-5
-  - topics/free-code-camp/responsive-web-design/2-basic-css
-  - topics/what-to-put-in-your-cv
-  - topics/web-frontend/intro-to-web-design/
+    - topics/free-code-camp/responsive-web-design/1-basic-html-and-html-5
+    - topics/free-code-camp/responsive-web-design/2-basic-css
```
Comment (ry-oc) [inline content/projects/first-personal-website/_index.md:9]: Why are you editing the spacing?

File: content/projects/first-personal-website/_index.md [hunk 3]
```diff
@@ -2,19 +2,19 @@
...
+    - topics/what-to-put-in-your-cv
+    - topics/web-frontend/intro-to-web-design/
```
Comment (ry-oc) [inline content/projects/first-personal-website/_index.md:11]: Why are you editing the spacing?

File: content/projects/first-personal-website/_index.md [hunk 4]
```diff
@@ -2,19 +2,19 @@
...
 ready: true
 story_points: 3
 submission_type: link
 tags:
-- html
-- css
+  - html
```
Comment (ry-oc) [inline content/projects/first-personal-website/_index.md:16]: Why are you editing the spacing?

**Review by ry-oc — CHANGES_REQUESTED** [review]
Please see comments and respond to them, also you have merge conflicts to resolve.

In order to resolve these, you need to update your forked develop with the original then merge the updated develop into your branch.

Let me know if you have issues with this and I will give you a hand.

File: content/projects/first-personal-website/_index.md (hunk 1 above)
Comment (KhanyiMM) [reply content/projects/first-personal-website/_index.md:5]: All I did was just add the additional link at the bottom and it seems the added spaces automatically occurred in each line. I will rectify that as well as the merge conflicts.

**Review by ry-oc — APPROVED** [review]

**Review by sheenarbw — CHANGES_REQUESTED** [review]
If you are reformatting links, then you might as well do it the markdown way. 

Take a look at the bottom of this page: content/projects/github-api-consume/_index.md

**Review by ry-oc — APPROVED** [review]
looks good

**Pull request merged**